services/provider/proto/provider.proto

```diff
@@ -13,8 +13,8 @@ service OCSProvider {
   rpc OnboardConsumer (OnboardConsumerRequest)
   returns (OnboardConsumerResponse) {}
   // GetStorageConfig RPC call to generate the json config for connecting to storage provider cluster
-  rpc GetStorageConfig(StorageConfigRequest)
-  returns (StorageConfigResponse){}
+  rpc GetStorageClientConfig(StorageClientRequest)
```

Comment:
pls recheck in client-op, this is still being used, only some of the info that it is sending isn't being used.

however, I like being explicit in the naming and so for backwards compat, create a new RPC and link old to new RPC.

this could change the implementation, will review after the change.

Reply:
Just looked at the client operator code and noticed that RPC is being used in 2 places to retrieve mon endpoints. I will add a new RPC.